webpack/webpack.config.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-undef */
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/index.js',
+  devServer: {
+    static: './dist',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: 'bundle.js',
+    path: path.join(__dirname, 'dist'),
+    clean: true,
+  },
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        use: ['style-loader', 'css-loader'],
+      },
+    ],
+  },
+  optimization: {
+    runtimeChunk: 'single',
+  },
+};
```

Comment:
- [ ] Kindly check the correct configuration for your webpack because when I run `npm start or npm run build`, it gives these errors.

- [ ] Also in your project there are two `package.json` files, please delete unnecessary files and add `index.js` file in the correct path. you can refer to the correct configuration for webpack from [here](https://webpack.js.org/guides/getting-started/#basic-setup)
- [ ] Kindly also add instruction for how to run the project using webpack in the `readme` 

| `npm start`  | `npm run build` |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/180222360-cfc8d8af-cac9-410a-b6aa-f16d28ee2f23.png)  | ![image](https://user-images.githubusercontent.com/53402600/180222494-b9d45cc5-34d1-47b6-8838-4dc567ca44cb.png) |